ofl/notoserifkhitansmallscript/METADATA.pb

```diff
@@ -24,7 +24,7 @@ is_noto: true
 sample_text {
   masthead_full: "𘭔𖿤𘭢 𘲽𘱸"
   masthead_partial: "𘰷𘭴"
-  styles: "𘰷𘭴 𘰝𘲥𘲦 𘲲𘯴𘱆 𘳍𘰄 𘰴𘳍𘰄 𘬜𘭞𘰷 𘬜𘭞𘰷 𘰷𘭞𘭟 𘰷𘭞𘭟 𘱓𘮯𘮡"
+  styles: "𘰷𘭴 𘰝𘲥𘲦 𘲲𘯴𘱆 𘳍𘰄 𘰴𘳍𘰄 𘬜𘭞𘰷"
```

Comment:
Is this enough? `𘰷𘭴 𘰝𘲥𘲦 𘲲𘯴𘱆 𘳍𘰄` might be sufficient?

Reply:
Addressed in https://github.com/google/fonts/pull/6586/commits/d2fd7df8cd84962a7671cfb9b212de0d036fabfd. Thanks Dave!